docs/csharp/index.md

```diff
@@ -60,6 +60,6 @@ There are several sections in the C# Guide. You can read them in order, or jump
 
 ## See also
 
-- [Getting Started with Visual C# and Visual Basic](/visualstudio/ide/getting-started-with-visual-csharp-and-visual-basic)
+- [Getting Started with Visual C# and Visual Basic](/visualstudio/ide/quickstart-visual-basic-console)
```

Comment:
Since the edited file is in csharp folder, I suggest the article to be the csharp one. I also edited the link text.

```suggestion
- [Getting Started with Visual C#](/visualstudio/ide/quickstart-csharp-console)
```